languages/cpp/ExampleUse.md

```diff
@@ -1,21 +1,26 @@
-# EXAMPLES
+# Examples
 
+## Prerequisites
 
-## PREREQUISITES
+### Bitwarden Libraries
+
+Have the two Bitwarden libraries at the same path:
 
-### BITWARDEN Libraries
-One should have two libraries at the same path:
 - `BitwardeClient`
```

Comment:
```suggestion
- `BitwardenClient`
```

Reply:
Great catch, updated in https://github.com/bitwarden/sdk/pull/854/commits/eec019b5c8623ee1902f53c7804e131c0b405445